test/get-player-opts.test.js

```diff
@@ -82,7 +82,7 @@ test('getPlayerOpts() with advertising', (t) => {
 
   t.equal(actual.aspectratio, '1:1', 'it sets the aspect ratio properly');
   t.equal(actual.playlist, mockPlaylist, 'it sets the playlist property to the supplied playlist');
-  t.equal(actual.mute, false, 'it sets the mute property to false');
+  t.equal(actual.mute, undefined, 'it sets the mute property to undefined');
```

Comment:
Hey this is great! Only question is should we make the test message more along the lines of `it doesn't set the mute property`? I think that's what's really happening, right?

Reply:
@danmakenoise sure, makes sense. Don't want to sound rude or anything, but as this blocks me from going live with a huge project, can we merge it today, please? :)
Please let me know if I can do anything to speed this up.

Reply:
@nirarazi, i'll go ahead and update the message and merge in shortly.

Reply:
@danmakenoise THANK YOU!!